CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to ideasbox.lan
+
+Do you like to make the ideasbox.lan project better? Welcome! This document
```

Comment:
I tried to adopt some friendly tone here... Feel free to comment about the words (not sure about my english language) or the tone itself.